gulpTasks/definition.js

```diff
@@ -88,4 +90,20 @@ gulp.task('internalDefs', function() {
   });
 });
 
-gulp.task('validateDefs', shell.task(['node node_modules/typescript/bin/tsc --noEmit ./bin/ts/CoveoJsSearch.d.ts']));
+gulp.task('validateDefs', ['validateTSV1', 'validateTSV2']);
+
+gulp.task('installTSV1', done => {
+  const version = '1.8.10';
+  async.waterfall([
```

Comment:
Would it be possible to use `async/await` or a `Promise` rather than the `async` lib? For the promise, either by chaining `tvm` methods if they return promises, or maybe something like:
```
new Promise(resolve =>  tvm.install(version, resolve))
.then(() => tvm.use(version, done))
```